### PR TITLE
Post-jam changes

### PR DESCRIPTION
**Pull request by thearst3rd**
Until the jam voting/playing time is up, I want the repository to reflect the final jam submission. Any changes we make in the meantime should target this branch, which will be merged after the jam is over.

### CODE REVIEW

**Comment by jokertyf** [comment]
Since the jam is officially over, time to merge this in?

**Comment by thearst3rd** [comment]
Good idea, lets merge it in and release v1.1

File: src/states/Game.gd [hunk 1]
```diff
@@ -62,20 +63,56 @@ func _on_player_move(grid_coords: Vector2):
...
+	var error := get_tree().change_scene("res://src/states/Menu.tscn")
+	assert(not error)
 
 
 func _on_OptionsButton_pressed() -> void:
-	$PauseMenu/ColorRect.hide()
-	$OptionsMenu/ColorRect.show()
-	$OptionsMenu/ColorRect/C/V/BackButton.grab_focus()
+	get_tree().paused = true
+	options_return_to_game = true
```
Comment (jokertyf) [inline src/states/Game.gd:105]: Not in love with this variable name, but I can't think of a better one.

File: src/states/Game.gd [hunk 2]
```diff
@@ -62,20 +63,56 @@ func _on_player_move(grid_coords: Vector2):
...
-	$PauseMenu/ColorRect/C/V/Buttons/OptionsButton.grab_focus()
+func _on_PauseMenu_OptionsButton_pressed() -> void:
+	options_return_to_game = false
+	$CanvasLayer/PauseMenu.hide()
+	$CanvasLayer/OptionsMenu.show_menu()
+
+
+func _on_RestartButton_pressed() -> void:
+	var error := get_tree().reload_current_scene()
+	assert(not error)
```
Comment (jokertyf) [inline src/states/Game.gd:118]: Probably just call this restart, and have the menu restart button/r key just call this function.

File: src/states/Game.gd (hunk 1 above)
Comment (thearst3rd) [reply src/states/Game.gd:105]: At least it causes the game to not crash LUL

I agree it's not great, but I also don't think it's much of an issue

File: src/states/Game.gd (hunk 2 above)
Comment (thearst3rd) [reply src/states/Game.gd:118]: As in, rename the function? Probably not a bad idea

**Review by kdreese — APPROVED** [review]

**Review by jokertyf — APPROVED** [review]

**Pull request merged**